procedures/katello/branch.md.erb

```diff
@@ -55,7 +55,11 @@
 
 ## Release Packager
 
-* [ ] Generate a release GPG key
+* [ ] In foreman-packaging rpm/<%= short_version %> branch
+  * [ ] Edit `katello-repos/katello.repo` to set `gpgcheck=1`
```

Comment:
This won't be needed after https://github.com/theforeman/foreman-packaging/pull/4206

Reply:
which is now merged